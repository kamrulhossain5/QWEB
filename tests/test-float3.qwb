function void testfloat(float a, float b){
  printf(a + b);
  printf(a - b);
  printf(a * b);
  printf(a / b);
  printb(a == b);
  printb(a == a);
  printb(a != b);
  printb(a != a);
  printb(a > b);
  printb(a >= b);
  printb(a < b);
  printb(a <= b);
}

function int main(){
  float c;
  float d;

  c = 25.0;
  d = 5.12345;

  testfloat(c, d);
  testfloat(d, d);

}
