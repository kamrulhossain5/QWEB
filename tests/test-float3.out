30.1234
19.8766
128.086
4.87952
0
1
1
0
1
1
0
0
10.2469
0
26.2497
1
1
1
0
0
0
1
0
1
